categories_map.py

```diff
@@ -1632,6 +1633,7 @@
     "upgrade": [
         "CalibTracker/SiPhase2TrackerESProducers",
         "CondFormats/HGCalObjects",
+        "Geometry/HGCalMapping",
```

Comment:
Please move this downwards to keep the alphabetic order.

Reply:
Moved it down so it should be in alphabetic order now